### ISSUE TITLE
Can not start Weles server

### ISSUE THREAD

**Issue by Shadasviar** [SamsungSLAV/weles#3]
I built Weles server with Docker from `master` branch from commit `0473c47d1dc629991911ba631e51b12048da788f` and got binary named `weles` in `bin` directory.
When I try run it as below:
```
./weles --boruta-address "http://127.0.0.1:8487" --host 127.0.0.1 --port 5010
```
I get error:
```
2019/02/02 23:26:42 failed to serve the APIlisten tcp: lookup linux on 8.8.8.8:53: no such host
```
I get the same error when run `weles` without parameters and with any other combination of parameters I tried.

-------------------
System parameters:
Operating System: openSUSE Tumbleweed 20190125
KDE Plasma Version: 5.14.5
Qt Version: 5.12.0
KDE Frameworks Version: 5.54.0
Kernel Version: 4.20.2-1-default
OS Type: 64-bit
Processors: 8 × Intel® Core™ i7-4710HQ CPU @ 2.50GHz
Memory: 7.7 GiB of RAM

**Comment by Michcioperz** [comment]
I can't reproduce that in my environment, but I do wonder why it tries to lookup "linux" on DNS.

**Comment by Michcioperz** [comment]
I don't have the debugger handy right now, but from strace logs it seems that `weles` binary reads `/etc/hosts`, `/etc/nsswitch.conf`, `/etc/mdns.allow` and `/etc/resolv.conf`.

I don't quite see either of these names in our dependencies, which would suggest that this comes from stdlib `net` package.

Does your hostname happen to be `linux`? (That still doesn't explain to me why it tries to look it up instead of just using 127.0.0.1)

**Comment by Shadasviar** [comment]
@michcioperz, yes, my hostname is `linux`. I attached here files you listed in your answer from my machine (/etc/mdns.allow does not exists):
[hosts.txt](https://github.com/SamsungSLAV/weles/files/2825975/hosts.txt)
[nsswitch.conf.txt](https://github.com/SamsungSLAV/weles/files/2825977/nsswitch.conf.txt)
[resolv.conf.txt](https://github.com/SamsungSLAV/weles/files/2825979/resolv.conf.txt)

And strace log of failed running:
[strace.log](https://github.com/SamsungSLAV/weles/files/2826003/strace.log)

**Comment by aalexanderr** [comment]
i wasnt able to reproduce it either :( could you try building on the host machine rather than docker?

**Comment by Michcioperz** [comment]
I don't think it's a problem with Docker, I built with Docker.

The question is why does it try to resolve the hostname (and fails because it's neither in /etc/hosts nor in DNS).

**Comment by Shadasviar** [comment]
@aalexanderr, I built it locally on my machine but problem still stay

**Comment by Shadasviar** [comment]
The way to reproduce on `VirtualBox`:
1. Download x86_64 image of `OpenSUSE Tumbleweed`: https://software.opensuse.org/distributions/tumbleweed (For 08.01.2019 it was 20190205 build)
2. Install it with graphical mode on `VirtualBox`, the DE which was used for reproduce is KDE (check it during installation process when it ask).
3. Build and run `Weles` in manual way.

**Comment by Shadasviar** [comment]
Build from latest buildable commit from `server_fix` branch from review didn't resolve problem

**Comment by aalexanderr** [comment]
I managed to reproduce it with latest opensuse image run in virtualbox. Will try to fix it in free time (not so much of it now)
